change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Package-lock updates are `none` changes
```suggestion
  "type": "none",
```

---

Looks like these all got changed back?

---

My bad, fixed.